### DISCUSSION TITLE
[BUG]默认站点设置无效

### DISCUSSION THREAD

**Discussion by QiYueYiya** [1Panel-dev/1Panel#6635] [Q&A]
### 联系方式

qiyueyi0215@qq.com

### 1Panel 版本

v1.10.2-lts

### 问题描述

默认站点我设置为 ariang 网站，但是每次访问未绑定 ip 都会被定向到alist网站，这两个网站都是使用的 10443 端口，只是用域名来区分访问，例如我访问 https://172.16.0.3:10443 ，此时应该被定向到 ariang 而不是 alist，但跳转的确实 alist，查看网站日志也只有 alist 有访问记录，当我停止 alist 网站后，则无法通过 https://172.16.0.3:10443 进行访问

### 重现步骤

![默认站点](https://github.com/1Panel-dev/1Panel/assets/64852147/d78bf31f-2eaf-457c-9fd2-89ebc9ab281d)
![ariang配置](https://github.com/1Panel-dev/1Panel/assets/64852147/d03bcab2-d987-4d29-a788-771908337e9b)
![alist配置](https://github.com/1Panel-dev/1Panel/assets/64852147/64b9b8f2-02a8-4da5-84b8-8d83b9c6472a)
![ariang日志](https://github.com/1Panel-dev/1Panel/assets/64852147/e713997e-6012-4a9a-a206-ea41f59c5608)
![alist日志](https://github.com/1Panel-dev/1Panel/assets/64852147/d63fe942-05a8-43b8-94d9-fc6dd560c5b7)

**Comment by zhengkunwang223** [comment]
我们先复现一下

**Comment by zhengkunwang223** [comment]
当前限制设置默认站点只能为 Openresty 的 HTTP 和  HTTPS 端口的网站
参考了其他面板 貌似也没有这个功能
如果要实现 可以需要把你这个网站所有的端口都设置为 default_server 
可以讨论一下 是否需要这么做
